ConfigCI/ViewController.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ViewController.swift
+//  ConfigCI
+//
+//  Created by Khoa Vo T.A. on 10/26/20.
+//  Copyright © 2020 Khoa Vo T.A. All rights reserved.
+//
+
+import UIKit
+
+final class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/waynelevi2607/ConfigCI/blob/37c45ea9423aa49167b51dd9c3eedc2156426f63/ConfigCI/ViewController.swift#L15"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>ViewController.swift:15</code></td>
    </tr>
  </tbody>
</table>